package.json

```
@@ -1,20 +1,21 @@
 {
   "name": "@metamask/etherscan-link",
-  "version": "1.1.0",
+  "version": "1.2.0",
   "description": "A library for generating etherscan links.",
-  "main": "index.js",
+  "main": "dist/index.js",
   "publishConfig": {
     "registry": "https://registry.npmjs.org/",
     "access": "public"
   },
   "files": [
-    "account-link.js",
-    "explorer-link.js",
-    "index.js",
-    "prefix-for-network.js"
+    "dist/account-link.js",
+    "dist/explorer-link.js",
+    "dist/index.js",
+    "dist/prefix-for-network.js"
```

We should change this to the entire `dist` folder. At the moment, this won't publish e.g. the type declarations.
```suggestion
    "dist"
```